portfolio-stylesheet.css

```diff
@@ -0,0 +1,125 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+}	
+
+/*@font-face {
+	font-family: "DIN Next LT";
+	src: url(DINNextLTPro-Regular.woff);
+}*/
+
+.navBar {
+	display: flex;
+	background-color: #FFFFFF;
+	padding: 5px;
+	border-bottom: 1px solid #dbdbdd;
+	justify-content: space-between;
+	align-items: center;
+	margin: 0 auto;
+	overflow: hidden;
+	position: sticky;
+	top: 0;
+	width: 100%;
+}
+
+nav {
+	width: 33.3%;
+}
+
+nav a {
+	color: #000000;
+	text-decoration: none;
+	padding-left: 10px;
+	padding-right: 10px;
+	font-size: 12.5px;
+	margin-left: 10px;
+}
+
+.navBar a:hover {
+	color: #5fd5ca;
+}
+
+h1 {
+	font-size: 30px;
+	width: 33.3%;
+	text-align: center;
+}
+
+.right {
+	width: 33.3%;
+}
+
+header {
+	padding: 380px 0;
+	width: 100%;
+	background-image: url('header.jpg');
```

Comment:
the header on my screen is white, not sure what the header img is, possibly need to git add/commit the images?